test/go-report-card-test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1
+FROM public.ecr.aws/bitnami/golang:latest
```

Comment:
Any reason to use latest over a fixed version? One benefit to the fixed version (1.16) would be easier debugging if these start to fail after an upgrade

Reply:
yeah that's true, I left these latest because I don't really care what version of go they use and don't really want to have to update these every time we increment the go version. The actual application container is pinned since we want to control that better. So I could go either way on this one...

Reply:
Right, I'm of the mind that we just peg it to the version our app is running, but agreed can go either way. I trust your judgement :D  

Some other deets:

* [go-report-card](https://github.com/gojp/goreportcard/blob/master/Dockerfile#L1) does not fetch latest golang image, but specific version (currently 1.16.3)
* `golicense` hasn't updated since 1.13 and doesn't have a Dockerfile

Reply:
yeah... dependencies are unfortunate but go's compatibility "promise" should help us here.